mindsdb/interfaces/model/model_controller.py

```diff
@@ -213,3 +248,118 @@ def import_predictor(self, name: str, payload: json, company_id: int) -> None:
             fp.write(predictor_binary)
 
         self.fs_store.push()
+
+    def process_create_statement(self, statement, database_controller, handler_controller):
+        # TODO use database_controller handler_controller internally
+
+        project_name = statement.name.parts[0]
+        model_name = statement.name.parts[1]
+
+        target = statement.targets[0].parts[-1]
+
+        # get data for learn
+        data_integration_id = None
+        fetch_data_query = None
+        if statement.integration_name is not None:
+            fetch_data_query = statement.query_str
+            integration_name = statement.integration_name.parts[0]
+
+            databases_meta = database_controller.get_dict()
+            data_integration_meta = databases_meta[integration_name]
+            # TODO improve here. Suppose that it is view
+            if data_integration_meta['type'] == 'project':
+                data_integration_id = handler_controller.get(name='views')['id']
+            else:
+                data_integration_id = data_integration_meta['id']
+
+        problem_definition = {'target': target}
+
+        label = None
+        if statement.using is not None:
+            label = statement.using.pop('tag', None)
+
+            problem_definition['using'] = statement.using
+
+        if statement.order_by is not None:
+            problem_definition['timeseries_settings'] = {
+                'is_timeseries': True,
+                'order_by': str(getattr(statement, 'order_by')[0])
+            }
+            for attr in ['horizon', 'window']:
+                if getattr(statement, attr) is not None:
+                    problem_definition['timeseries_settings'][attr] = getattr(statement, attr)
+
+            if statement.group_by is not None:
+                problem_definition['timeseries_settings']['group_by'] = [str(col) for col in statement.group_by]
+
+        join_learn_process = False
+        if 'join_learn_process' in problem_definition.get('using', {}):
+            join_learn_process = problem_definition['using']['join_learn_process']
+            del problem_definition['using']['join_learn_process']
+
+        return dict(
+            model_name=model_name,
+            project_name=project_name,
+            data_integration_id=data_integration_id,
+            fetch_data_query=fetch_data_query,
+            problem_definition=problem_definition,
+            join_learn_process=join_learn_process,
+            label=label
+        )
+
+    def create_predictor(self, statement, ml_handler, company_id: int):
```

Comment:
Please:
1. provide docstrings in google styleguide format https://google.github.io/styleguide/pyguide.html#s3.8-comments-and-docstrings.
2. provide type of input parameters. you may do in direct in function signature or in the doc strings.